### PR TITLE
Client dashboard

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Trello Card Link
https://trello.com/c/McTu5r6t/1626-make-sure-client-dashboard-is-wai

### Description
Supports client dashboard

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

Manually tested:
* claim flows
* send flows
* receive flows

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/client-dashboard

File: src/services/AudiusBackend.js [hunk 1]
```diff
@@ -36,7 +36,11 @@ const WEB3_NETWORK_ID = process.env.REACT_APP_WEB3_NETWORK_ID
 const ETH_REGISTRY_ADDRESS = process.env.REACT_APP_ETH_REGISTRY_ADDRESS
 const ETH_PROVIDER_URLS = process.env.REACT_APP_ETH_PROVIDER_URL.split(',')
 const ETH_TOKEN_ADDRESS = process.env.REACT_APP_ETH_TOKEN_ADDRESS
+const ETH_RELAYER_PUBLIC_KEY = process.env.REACT_APP_ETH_RELAYER_PUBLIC_KEY
 const ETH_OWNER_WALLET = process.env.REACT_APP_ETH_OWNER_WALLET
+const COM_STOCKURL = process.env.REACT_APP_COMSTOCK_URL
```
Comment (piazzatron) [inline src/services/AudiusBackend.js:41]: COM_STOCKURL

File: src/services/AudiusBackend.js [hunk 2]
```diff
@@ -2363,6 +2369,96 @@ class AudiusBackend {
...
+
+  /**
+   * Make the claim for the distribution
+   * NOTE: if the claim was already made, the response will 500 and error
+   * @returns {Promise<boolean>} didMakeClaim
+   */
+  static async makeDistributionClaim() {
+    await waitForLibsInit()
+    const wallet = audiusLibs.web3Manager.getWalletAddress()
+    if (!wallet) return
```
Comment (piazzatron) [inline src/services/AudiusBackend.js:2399]: Super minor but all these fns have an undefined return that makes them tricky to type properly, can we return either false or null in the no wallet case?

File: src/services/wallet-client/WalletClient.ts [hunk 3]
```diff
@@ -16,44 +16,46 @@ class WalletClient {
...
-      setTimeout(() => {
-        resolve(audioToWei('100000' as StringAudio))
-      }, 1500)
-    })
+    try {
+      const balance = await AudiusBackend.getBalance()
+      return balance as BNWei
+    } catch (err) {
+      console.log(err)
+      return new BN('0') as BNWei
```
Comment (piazzatron) [inline src/services/wallet-client/WalletClient.ts:24]: Nit: would be v nice to have a constant ZeroBNWei that we can just import from the wallet client or something

**Review by piazzatron — APPROVED** [review]
GJ guys ✨

**Pull request merged**